### PR TITLE
RHEL bump from 8.3 --> 8.4 in RNs (4.8 version)

### PR DESCRIPTION
**Pull request by bobfuru**
RHCOS got a bump from using RHEL 8.3 to RHEL 8.4. Currently, the OCP 4.8 RNs state that 4.8 is using RHEL 8.3. 
xref: http://mailman-int.corp.redhat.com/archives/coreos-devel/2021-August/msg00000.html

This PR updates this, and change management is also required.

Preview link:
https://deploy-preview-35265--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-8-rhcos-rhel-8-4-packages

Relates to https://github.com/openshift/openshift-docs/pull/35261
Change management email sent Aug. 5

### CODE REVIEW

**Comment by bobfuru** [comment]
@travier and @miabbott PTAL for SME and QE review, please.

**Comment by miabbott** [comment]
Yikes!  We never used 8.3 in 4.8, so this is a very necessary change.

LGTM

**Comment by travier** [comment]
> Yikes! We never used 8.3 in 4.8, so this is a very necessary change.

The release only mention 8.3 for 4.7 so I think they were correct.

Same comment as https://github.com/openshift/openshift-docs/pull/35261#issuecomment-894171155

**Comment by bobfuru** [comment]
> The release notes only mention 8.3 for 4.7 so I think they were correct.

Yes, they mention only that 4.7 uses 8.3, so that part is good.

I'll update both PRs to indicate the 4.7 z-stream release when use of 8.4 begins but will hold off in merging until that one (4.7.24) lands.

**Comment by bobfuru** [comment]
@travier @miabbott - Same comment as in https://github.com/openshift/openshift-docs/pull/35261#issuecomment-899827517 - please confirm if this wording looks good for 4.8 RNs once 4.7.24 is GA.

**Review by travier — APPROVED** [review]
LGTM

**Comment by bobfuru** [comment]
[OCP 4.7.24](https://docs.openshift.com/container-platform/4.7/release_notes/ocp-4-7-release-notes.html#ocp-4-7-24) is GA now. With @miabbott on PTO, I plan to merge this today after peer review because it has been reviewed twice by Eng. and was already checked by QE previously.

File: release_notes/ocp-4-8-release-notes.adoc
Comment (codyhoag) [inline release_notes/ocp-4-8-release-notes.adoc:61]: Extra space after `driver updates.`

**Review by codyhoag — COMMENTED** [review]
Just one tiny nit; looks good otherwise!

File: release_notes/ocp-4-8-release-notes.adoc
Comment (bobfuru) [reply release_notes/ocp-4-8-release-notes.adoc]: Yikes good catch! Fixed.

**Pull request merged**
